tests/utils/common/oidc.go

```diff
@@ -109,15 +109,18 @@ func ParseIssuerURLFromCommand(command string) string {
 
 // Extract oidc provider from the 'OIDC Endpoint URL' field of `rosa describe cluster`
 func ExtractOIDCProviderFromOidcUrl(urlString string) (string, error) {
+	var oidcProvider string
+	urlString = strings.TrimSpace(strings.Split(urlString, " ")[0])
 	parsedURL, err := url.Parse(urlString)
 	if err != nil {
 		return "", err
 	}
 	host := parsedURL.Host
 	path := strings.TrimPrefix(parsedURL.Path, "/")
-	fmt.Printf("The past host and path is %s and %s\n", host, path)
-	oidcProvider := fmt.Sprintf("%s/%s", host, path)
-	oidcProvider = strings.Split(oidcProvider, " ")[0]
-
+	if path != "" {
+		oidcProvider = fmt.Sprintf("%s/%s", host, path)
+	} else {
+		oidcProvider = host
+	}
```

Comment:
I'd like to write it like this:
```
oidcProvider = host
if path != "" {
  oidcProvider = fmt.Sprintf("%s/%s", host, path)
}

Reply:
I don't know what difference with the if-else way, please let me know why you think that's better. 
However there is no impact, I will update even you have no reason why you suggest this?

Reply:
The way I recommend could improve the readability.